mypy.ini

```diff
@@ -1,40 +0,0 @@
-[mypy]
-python_version = 3.7
-check_untyped_defs = true
-disallow_subclassing_any = true
-disallow_incomplete_defs = true
-no_implicit_optional = true
-pretty = true
```

Comment:
It seems like we're changing a lot of the settings here. What's the motivation? Many of these settings seem useful.